packages/config/config/devices/0x0330/zv9001k12-dim-z5.json

```diff
@@ -0,0 +1,59 @@
+// Sunricher ZV9001K12-DIM-Z5
+// 5 Channel Dimmer Remote Control
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV9001K12-DIM-Z5",
+	"description": "5 Channel Dimmer Remote Control",
+	"devices": [
+		{
+			"productType": "0x0300",
+			"productId": "0xb301"
+		},
+		{
+			"productType": "0x0301",
+			"productId": "0xb301"
+		},
+		{
+			"productType": "0x0302",
+			"productId": "0xb301"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Launch 2",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Launch 3",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Launch 4",
+			"maxNodes": 5
+		},
+		"6": {
+			"label": "Launch 5",
+			"maxNodes": 5
+		}
+	},
```

Comment:
These devices support Association Group Info CC and Z-Wave+, so associations can be auto-discovered. Unless the auto-discovered labels are worse than this, the entire block here is unnecessary.